samples/highcharts/demo/combo-regression/demo.js

```diff
@@ -1,83 +1,203 @@
+function generateChart(dataset) {
+    Highcharts.chart('container', {
+        title: {
+            text: 'Scatter plot with trend line'
+        },
+        xAxis: {
+            min: -3.5,
+            max: 5.5
```

Comment:
If possible, avoid data-dependent hard coding. It will cause unnecessary friction when users apply their own data set.
```suggestion
            minPadding: 0.1,
            maxPadding: 0.1
```